features/kuzzle.feature

```diff
@@ -604,6 +604,7 @@ Feature: Kuzzle functional tests
     Then I am able to perform a scrollUsers request
     Then I delete the user "user2-id"
     Then I search for {"ids":{"type": "users", "values":["#prefix#useradmin-id"]}} and find 1 users matching {"_id":"#prefix#useradmin-id","_source":{"name":{"first":"David","last":"Bowie"}}}
+    Then I revoke all tokens of the user "useradmin-id"
     When I log in as useradmin:testpwd expiring in 1h
```

Comment:
Shoudn't we test the the user's token are actually revoked?

something like:
- log in as user
- revoke user's token
- call any action
- get an error (which one btw?)

Probably deserves a dedicated scenario.